OracleGoldenGate/21c/Dockerfile

```diff
@@ -0,0 +1,27 @@
+## Copyright (c) 2021, Oracle and/or its affiliates.
+FROM          oraclelinux:8-slim
```

Comment:
We do not recommend the use of `oraclelinux:8-slim` for anything except microservices. Please rebase this to use `oraclelinux:8` instead.